src/main/java/se/icus/mag/statuseffecttimer/mixin/StatusEffectTimerMixin.java

```diff
@@ -55,8 +56,12 @@ private void renderDurationOverlay(MatrixStack matrices, CallbackInfo c) {
 					}
 
 					String duration = getDurationAsString(statusEffectInstance);
-					int length = client.textRenderer.getWidth(duration);
-					drawStringWithShadow(matrices, client.textRenderer, duration, x + 13 - (length / 2), y + 14, 0x99FFFFFF);
+					int durationLength = client.textRenderer.getWidth(duration);
+					drawStringWithShadow(matrices, client.textRenderer, duration, x + 13 - (durationLength / 2), y + 14, 0x99FFFFFF);
```

Comment:
Do you think you could extract the `0x99FFFFFF` color code into a constant? It was ugly as it was to have it hard-coded like that, and duplicating it makes it worse. :-)